### ISSUE TITLE
Update/Fix running hubs page in docs

### ISSUE THREAD

**Issue by sgibson91** [2i2c-org/infrastructure#1009]
I understand this change as we will now render this part without the `Name` column - at the same time, this reduced the complexity by not entangling configuration significantly.

I support this change, but I think it would be nice to retain this. Perhaps by adding a field to cluster.yaml - which indeed may need to be duplicated as well. Hmm... I propose we move onwards but add a reference to this comment in the todo-list associated with this PR for later consideration.

![image](https://user-images.githubusercontent.com/3837114/153643158-8de2b711-47be-4c03-96bd-596e60a30f96.png)

_Originally posted by @consideRatio in https://github.com/2i2c-org/infrastructure/pull/985#discussion_r804882713_

**Comment by choldgraf** [comment]
@sgibson91 are there other issues we need to resolve first before addressing this one? You mentioned changing up some other configuration that'll also break this logic, so i figure we should get all of that work done first before we un-break it.

**Comment by sgibson91** [comment]
Yep, it's moving the support chart config out of the `cluster.yaml` files. That's being tracked in #879
